change/@fluentui-react-dialog-19f431ea-3879-40f6-bf2a-177d37d60310.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27367&commitId=0816eb2b2f495ff8cd3a602fa570191bba876925&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1899 screenshots unchanged, 0 screenshots with different dimensions and 4 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.scroll long content - Dark Mode.chromium.png|38| Changed|
|Dialog.scroll long content - High Contrast.chromium.png|45| Changed|
|Dialog.scroll long content - RTL.chromium.png|39| Changed|
|Dialog.scroll long content.chromium.png|39| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>